apps/condo/domains/ticket/utils/serverSchema/TicketDocumentGeneration/paidWorks.js

```diff
@@ -0,0 +1,195 @@
+
+async function getLocaleToPullFinanceInfoClient (locale, organization) {
+    let psrn = null, organizationAddress = null, iec = null
+
+    if (locale === 'ru') {
+        try {
+            ({ iec, psrn, organizationAddress } = await financeInfoClient.getOrganization(organization.tin))
+        } catch (error) {
+            logger.info({ msg: 'fall financeInfoClient when get organization by tin', organizationId: organization.id, tin: organization.tin, error: error })
+        }
+    }
+
+    return { iec, psrn, organizationAddress }
+}
+
+const changeUndefinedDataToBlanks = (documentDataInLine) => {
+    const metaInfoAboutLongWords = {
+        name: true,
+        address: true,
+    }
+    const chengedDocumentDataInLine = Object.fromEntries(Object.entries(documentDataInLine).map(([key, value]) => {
+        let changedValue = {}
+        for (let subValue in value) {
+            changedValue[subValue] = value[subValue] ? value[subValue] : metaInfoAboutLongWords.hasOwnProperty(subValue) ? '____________________________________' : '__________________'
+        }
+        return [key, changedValue]
+    }))
+    
+    return chengedDocumentDataInLine
+}
+
+const buildExportWordFile = async ({ task, documentData, locale, timeZone }) => {
+    const { id, ticket } = task
+
+    const { stream } = await buildExportFile({
+        templatePath: `./domains/ticket/templates/ticketDocumentGenerationTemplates/paidWorks/${locale}.docx`,
+        replaces: documentData,
+    })
+
+    return {
+        stream,
+        filename: `paid_works_ticket_${ticket.id}_${formatDateToTimezone(undefined, timeZone, 'DD_MM_YYYY')}.docx`,
+        mimetype: DOCX_FILE_META.mimetype,
+        encoding: DOCX_FILE_META.encoding,
+        meta: {
+            listkey: 'TicketDocumentGenerationTask',
+            id,
+        },
+    }
+}
+
+const generateTicketDocumentOfPaidWorks = async ({ task, baseAttrs, context, locale, ticket, organization }) => {
+    const { iec, psrn, organizationAddress } = await getLocaleToPullFinanceInfoClient(locale, organization)
+
+    const { format, timeZone: timeZoneFromUser } = task
+
+    const timeZone = normalizeTimeZone(timeZoneFromUser) || DEFAULT_ORGANIZATION_TIMEZONE
+    const printDate = dayjs().tz(timeZone).locale(locale)
+
+    const contact = ticket.contact ? await getByCondition('Contact', {
+        id: ticket.contact,
+        deletedAt: null,
+    }) : null
+
+    const employee = organization.id && ticket.executor
+        ? await getByCondition('OrganizationEmployee', {
+            organization: { id: organization.id },
+            user: { id: ticket.executor },
+            deletedAt: null,
+        })
+        : null
+
+    const invoices = await Invoice.getAll(context, {
+        ticket: { id: ticket.id },
+        deletedAt: null,
+        status_not: INVOICE_STATUS_CANCELED,
+    }, 'id currencyCode accountNumber rows { isMin toPay name count vatPercent } recipient { bankName bankAccount bic }', {
```

Comment:
why specify fields like that? Is it how it should be done? Why not `graphql` file?

Reply:
^^

Reply:
> why specify fields like that? Is it how it should be done? Why not `graphql` file?

Because that's how server utilities work now :) Now you should request only the fields you need